tests/whitelist.txt

```diff
@@ -4,9 +4,6 @@ integration.fileserver.test_fileclient
 integration.fileserver.test_roots
 integration.loader.test_ext_grains
 integration.loader.test_ext_modules
-integration.loader.test_globals
-integration.loader.test_interfaces
-integration.loader.test_loader
```

Comment:
I don't think unit tests are running in the test suite... possibly because I'm passing --names-file